src/main/resources/application.yml

```diff
@@ -1,21 +1,45 @@
+
+spring:
+  pulsar:
+#    client:
```

Comment:
I have this commented out because I was running against a local Pulsar in docker-compose.